blocklylib-core/src/main/assets/default/loop_blocks.json

```diff
@@ -63,7 +63,7 @@
       {
         "type": "field_variable",
         "name": "VAR",
-        "variable": null
+        "variable": "i"
```

Comment:
The original `null` is [how it is defined in web](https://github.com/google/blockly/blob/master/blocks/loops.js#L127). So if there is a difference in behavior, then the solution is elsewhere.

Reply:
Fixed!